### PR TITLE
Khajra/isr1

### PR DESCRIPTION
**Pull request by khajra**
Sample for image service raster from a url.

### CODE REVIEW

**Comment by khajra** [comment]
@ldanzinger Can you please review and make sure if everything looks good?  
@nandinirao Please look over it also.

File: ArcGISRuntimeSDKQt_CppSamples/Layers/RasterLayerService/RasterLayerService.cpp [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  // find QML MapView component
+  m_mapView = findChild<MapQuickView*>("mapView");
+  m_mapView->setWrapAroundMode(WrapAroundMode::Disabled);
+
+  // create an image service raster
+  ImageServiceRaster* imageServiceRaster = new ImageServiceRaster(QUrl("http://sampleserver6.arcgisonline.com/arcgis/rest/services/NLCDLandCover2001/ImageServer"), this);
+  // create a raster layer using the image service raster
+  m_rasterLayer = new RasterLayer(imageServiceRaster, this);
+  // add the raster layer to the map's operational layers
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Layers/RasterLayerService/RasterLayerService.cpp]: update comment for basemap

File: ArcGISRuntimeSDKQt_CppSamples/Layers/RasterLayerService/RasterLayerService.h [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  ~RasterLayerService();
+
+  static void init();
+
+  void componentComplete() Q_DECL_OVERRIDE;
+
+private:
+
+  Esri::ArcGISRuntime::Map* m_map = nullptr;
+  Esri::ArcGISRuntime::Basemap* m_basemap = nullptr;
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Layers/RasterLayerService/RasterLayerService.h]: this member isn't used

File: ArcGISRuntimeSDKQt_CppSamples/Layers/RasterLayerService/RasterLayerService.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "source" :  "RasterLayerService.qml",
+    "name" : "Raster layer (service)",
+    "otherSourceFiles" :  [
+        "RasterLayerService.h",
+        "RasterLayerService.cpp",
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Layers/RasterLayerService/RasterLayerService.json]: extra comma after `.cpp",`

File: ArcGISRuntimeSDKQt_CppSamples/Layers/RasterLayerService/RasterLayerService.qml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 1.4
+import Esri.Samples 1.0
+import Esri.ArcGISExtras 1.1
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Layers/RasterLayerService/RasterLayerService.qml]: not needed. scalefactor property not needed either.

File: ArcGISRuntimeSDKQt_CppSamples/Layers/RasterLayerService/main.cpp [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+#ifdef Q_OS_WIN
+#include <Windows.h>
+#endif
+
+#include "RasterLayerService.h"
+
+#define STRINGIZE(x) #x
+#define QUOTE(x) STRINGIZE(x)
+
+using namespace Esri::ArcGISRuntime;
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Layers/RasterLayerService/main.cpp]: not needed

File: ArcGISRuntimeSDKQt_QMLSamples/Layers/RasterLayerService/RasterLayerService.qml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 1.4
+import Esri.ArcGISRuntime 100.1
+import Esri.ArcGISExtras 1.1
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_QMLSamples/Layers/RasterLayerService/RasterLayerService.qml]: not needed. scaleFactor can be removed too

**Comment by khajra** [comment]
@nandinirao and @ldanzinger:  Changes are in.  Please take a look.

**Pull request merged**